heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -230,6 +230,15 @@ local function process_json(msg, json, parsed)
         update_field(msg.Fields, "normalizedChannel" , fx.normalize_channel(auc))
 
         -- The "telemetryEnabled" flag does not apply to this type of ping.
```

Comment:
Yeah, it's explaining why we don't set that field for appusage pings. It's used as a dimension in the "telemetry-release" dataset, which is why it's (arguably) comment-worthy.